src/App.jsx

```diff
@@ -14,33 +24,44 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button type="button" className="button is-info is-light">
-        Sort alphabetically
-      </button>
+function sortGoods(goods, { sortBy: criteria, reverse }) {
+  let sortedGoods;
 
-      <button type="button" className="button is-success is-light">
-        Sort by length
-      </button>
+  switch (true) {
+    case criteria === SORT_BY_NAME:
+      sortedGoods = goods.toSorted((a, b) => a.localeCompare(b));
+      break;
+    case criteria === SORT_BY_LENGTH:
+      sortedGoods = goods.toSorted((a, b) => a.length - b.length);
+      break;
 
-      <button type="button" className="button is-warning is-light">
-        Reverse
-      </button>
+    default:
+      sortedGoods = [...goods];
+  }
 
-      <button type="button" className="button is-danger is-light">
-        Reset
-      </button>
-    </div>
+  if (reverse) {
+    sortedGoods = sortedGoods.toReversed();
+  }
+
+  return sortedGoods;
+}
+
+export const App = () => {
+  const [sortBy, setSortBy] = useState(RESET_VALUE);
+  const [reverse, setReversed] = useState(REVERSE);
```

Comment:
```suggestion
  const [reversed, setReversed] = useState(REVERSE);
```
or even better isReversed and setIsReversed for boolean variable